README.md

```diff
@@ -1,4 +1,36 @@
-[![Build Status](https://travis-ci.org/datahq/data.js.svg?branch=master)](https://travis-ci.org/datahq/data.js)
 ## Data.js
+<p align="center">
+  <a href="https://datahub.io/">
+    <img alt="datahub" src="http://datahub.io/static/img/logo-cube.png" width="146">
+  </a>
+</p>
 
-Library for working with Dataset and File objects.
+<p align="center">
+  Library for working with Dataset and File objects.
+</p>
+
+[![Build Status](https://travis-ci.org/datahq/data.js.svg?branch=master)](https://travis-ci.org/datahq/data.js) [![Gitter](https://img.shields.io/gitter/room/frictionlessdata/chat.svg)](https://gitter.im/datahubio/chat) [![Dependency Status](https://david-dm.org/datahq/data.js/status.svg)](https://david-dm.org/datahq/data.js) [![devDependency Status](https://david-dm.org/datahq/data.js/dev-status.svg)](https://david-dm.org/datahq/data.js?type=dev)
+
+### Usage
```

Comment:
This should be a second level heading and i think the title should be `## Installation`

Do you need to make a note about which node versions are supported.